diff --git a/dist/server/services/utils/configuration/attributes.js b/dist/server/services/utils/configuration/attributes.js
index d58d109556bd240952a7cd92088e7aa12acfde39..1e91f43add19ba66460cbcb22ea16b953c7cf3f1 100644
--- a/dist/server/services/utils/configuration/attributes.js
+++ b/dist/server/services/utils/configuration/attributes.js
@@ -19,6 +19,7 @@ const SORTABLE_RELATIONS = [
     'manyToOne'
 ];
 const NON_LISTABLES = [
+    'component',
     'json',
     'password',
     'richtext',
